reports/tasks.py

```diff
@@ -76,6 +68,21 @@ def send_trial_instances_report(recipients=settings.TRIAL_INSTANCES_REPORT_RECIP
     end_of_last_month = (beginning_of_this_month - timedelta(days=1))
     beginning_of_last_month = end_of_last_month.replace(day=1)
 
+    # Get instances with an active betatestapplication,
+    # whose InstanceReference has an active OpenEdXAppServer
+    # and was created last month
+    instances = OpenEdXInstance.objects.exclude(
+        betatestapplication__isnull=True
+    ).filter(
+        ref_set__openedxappserver_set___is_active=True
+    ).filter(
+        ref_set__created__range=[
+            beginning_of_last_month.strftime("%Y-%m-%d"),
```

Comment:
@spokerman12, since this is just a time range, how much effort would it require to make this configurable instead of hard-coding it to the last completed month?

Reply:
It'd be easy, just adding more arguments to send_trial_instances_report.
But right now I don't think it's convenient to increase the scope as we still have tasks left; I'd be happy to do it on the next sprint, though.